### PR TITLE
Add windows10 steps to install docker through the command line

### PR DESCRIPTION
**Pull request by vinmazzi**
## Summary
Add to the documentation the command line to install docker on windows 10.

### CODE REVIEW

File: docs/src/quickstart/index.md [hunk 1]
```diff
@@ -21,25 +21,32 @@ or
...
+* Virtualization Technology (VT-x) enabled on the BIOS 
+
 ## {#qs-win-install} Installation
 
-Click the Windows icon, type `CMD`, and then click `Run as Administrator`.
+Click the Windows icon, type `Powershell`, and then click `Run as Administrator`.
 There are a number of commands that need to be run to get Windows ready for Reach.
 
-To install WSL on Windows, run:
+We want to download a tool from Microsoft's github called winget.
```
Comment (jeapostrophe) [inline docs/src/quickstart/index.md:31]: - GitHub
- `winget`

File: docs/src/quickstart/index.md [hunk 2]
```diff
@@ -21,25 +21,32 @@ or
...
+Winget is the official Windows package manager:
 
+To install Winget run:
 ``` cmd
-$ wsl --install
+cd c:\; mkdir winget_download; cd winget_download; 
+wget -uri https://github.com/microsoft/winget-cli/releases/download/v1.3.431/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle -outfile .\winget.msixbundle; .\winget.msixbundle;
```
Comment (jeapostrophe) [inline docs/src/quickstart/index.md:37]: Will these numbers change?

File: docs/src/quickstart/index.md [hunk 3]
```diff
@@ -21,25 +21,32 @@ or
...
+
 ## {#qs-win-install} Installation
 
-Click the Windows icon, type `CMD`, and then click `Run as Administrator`.
+Click the Windows icon, type `Powershell`, and then click `Run as Administrator`.
 There are a number of commands that need to be run to get Windows ready for Reach.
 
-To install WSL on Windows, run:
+We want to download a tool from Microsoft's github called winget.
+Winget is the official Windows package manager:
```
Comment (jeapostrophe) [inline docs/src/quickstart/index.md:32]: `winget`?

File: docs/src/quickstart/index.md [hunk 4]
```diff
@@ -48,13 +55,23 @@ In the Command Prompt window, run the following command to set the WSL version t
...
+``` cmd
+winget install docker.dockerdesktop
+```
+
+After installing docker you should reboot:
+``` cmd
+shutdown -d -t 0
+```
+
+After rebooting a WSL2 terminal will openup informing it is finishing the installation.
```
Comment (jeapostrophe) [inline docs/src/quickstart/index.md:69]: open up

File: docs/src/quickstart/index.md [hunk 5]
```diff
@@ -48,13 +55,23 @@ In the Command Prompt window, run the following command to set the WSL version t
...
+When the installation is finished it will request you to setup a user and password, make sure you don't skip this step
+
+Configuring docker:
```
Comment (jeapostrophe) [inline docs/src/quickstart/index.md:72]: Heading

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs/src/quickstart/index.md (hunk 1 above)
Comment (hamirmahal) [reply docs/src/quickstart/index.md:31]:
```suggestion
We want to download a tool from Microsoft's GitHub called `winget`.
```

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**